contents/data-display/timeline/simple-timeline/timeline-item.tsx

```diff
@@ -0,0 +1,56 @@
+import {
+  Box,
+  Card,
+  CardBody,
+  CardHeader,
+  Heading,
+  Badge,
+  Text,
+  ListItem,
+} from "@yamada-ui/react"
+import type { FC } from "react"
+
+interface TimelineItemProps {
+  date: string
+  title: string
+  description: string
+}
+
+export const TimelineItem: FC<TimelineItemProps> = ({
+  date,
+  title,
+  description,
+}) => (
+  <Card
+    as={ListItem}
+    variant="subtle"
+    position="relative"
+    maxW="xl"
+    bg="transparent"
+    pt="sm"
+    gap="sm"
+  >
+    <Badge position="absolute" top="0" left="md" colorScheme="primary">
+      {date}
+    </Badge>
+    <Box
+      position="absolute"
+      left="-2.05rem"
+      top="15%"
+      transform="translateY(-50%)"
```

Comment:
Would this be more appropriate?

```suggestion
      translateY="50%"
```

Reply:
It would not work because transform style isn't applied to the `<Center />` component by default.

Reply:
OK.